src/api/auth-api.js

```diff
@@ -71,6 +71,41 @@ export const authApi = {
                 }, err => reject(err)
             )
         })
-    }
+    },
+
+    forgotEmail: (email) => {
+        return new Promise ((resolve, reject) => {
+            httpServise.post(`${host}:${port}/forgot`, {email: email}).then(
+                res => {
+                    if (res.data.text) {
+                        resolve(res.data.text);
+                    } else reject({status: 401});
+                }, err => reject(err)
+            )
+        })
+    },
 
+    VerifyPasswordReset: (r) => {
+        return new Promise ((resolve, reject) => {
+            httpServise.get(`${host}:${port}/change/${r}`).then(
+                res => {
+                    if (res.data.text) {
+                        resolve(res.data.text);
+                    } else reject({status: 401});
+                }, err => reject(err)
+            )
+        })
+    },
+
+    ChangePassword: (password, passwordConf, resetToken) =>{
+        return new Promise ((resolve, reject) => {
+            httpServise.post(`${host}:${port}/change`, {password: password, passwordConf: passwordConf, resetToken: resetToken}).then(
```

Comment:
dont duplicate key and variable, you can use {password, passwordConf, resetToken}